### ISSUE TITLE
Allow to customize history length for plots

### ISSUE THREAD

**Issue by thomasnordquist** [thomasnordquist/MQTT-Explorer#150]
Plots may currently hold 500 data points.

What has to be done:
- [ ] Pre-process incoming data (so it does not have to be processed each render)
- [ ] One should be able to define a custom amount

**Comment by guillochon** [comment]
Yes please! 500 points only shows the past hour for some of my devices, particular power monitoring plugs that publish every 5 seconds or so.
